--- leveson041011amd1.txt	2013-09-24 12:56:20.000000000 +0100
+++ leveson041011amd1.txt.fixed	2013-09-25 19:18:11.000000000 +0100
@@ -118,15 +118,15 @@
 (+44) 207 404 1400                                                                                             London EC4A 2DY
                                                 Leveson Inquiry Initial Hearing                                                4 October 2011
 
-  1        So I move on having identified the issues which         1    MR JAY: Yes, we have.
-  2     others will adumbrate in advance, perhaps in greater       2    LORD JUSTICE LEVESON: Nobody has commented on the draft
-  3     detail, to deal with the documents protocol which has      3      funding protocol.
-  4     excited only one observation on my reckoning and this is   4    MR JAY: No.
-  5     uncontroversial really. In paragraph 15 of the             5    LORD JUSTICE LEVESON: Right, okay. So this is now "Comment
-  6     documents protocol which is in your bundle page 14D,       6      on briefing seminars".
-  7     sir, Associated Newspapers have suggested that we have     7    MR JAY: Yes, or the subjects covered by the briefing
-  8     a third category after the "or", what is going to be       8      seminars, as News International put it. So the
-  9     a second "or":                                             9      position, as you know, under Rule 11 is that there is
+  1        So I move on having identified the issues which          1   MR JAY: Yes, we have.
+  2     others will adumbrate in advance, perhaps in greater        2   LORD JUSTICE LEVESON: Nobody has commented on the draft
+  3     detail, to deal with the documents protocol which has       3     funding protocol.
+  4     excited only one observation on my reckoning and this is    4   MR JAY: No.
+  5     uncontroversial really. In paragraph 15 of the              5   LORD JUSTICE LEVESON: Right, okay. So this is now "Comment
+  6     documents protocol which is in your bundle page 14D,        6     on briefing seminars".
+  7     sir, Associated Newspapers have suggested that we have      7   MR JAY: Yes, or the subjects covered by the briefing
+  8     a third category after the "or", what is going to be        8     seminars, as News International put it. So the
+  9     a second "or":                                              9     position, as you know, under Rule 11 is that there is
  10        "Confirm in writing that no redactions in respect of    10     express provision for making opening statements. At the
  11     the documents or information provided are sought."         11     moment, it should probably be most appropriately made at
  12        That is an acceptable amendment and we will make        12     the commencement of the formal segment of the Inquiry
@@ -151,13 +151,13 @@
   4     about the practical proposals which are there suggested.  4     LORD JUSTICE LEVESON: Yes.
   5     I think it is Trinity Mirror, isn't it, who made those    5     MR JAY: So, in answer to News International's question,
   6     suggestions?                                              6       well they can say precisely what they like at that point
-  7   LORD JUSTICE LEVESON: Yes, but using their expertise from 7         and it need not be limited, of course, to the subjects
+  7   LORD JUSTICE LEVESON: Yes, but using their expertise from   7       and it need not be limited, of course, to the subjects
   8     another Inquiry.                                          8       covered by the briefings and seminars, it can be as wide
   9   MR JAY: So it was the MPA. It is page 108. I am grateful.   9       ranging as they see fit.
  10   LORD JUSTICE LEVESON: Thank you very much. Yes, it is      10          You also, it seems to us, have power, at any stage
  11     Eversheds.                                               11       of the Inquiry, to invite submissions from counsel to
  12   MR JAY: Yes.                                               12       the Inquiry or any core participants on any specific
- 13   LORD JUSTICE LEVESON: I have no doubt that members of the 13        issue.
+ 13   LORD JUSTICE LEVESON: I have no doubt that members of the  13       issue.
  14     Inquiry team will talk to solicitors who obviously have  14     LORD JUSTICE LEVESON: Let me just give you some thinking on
  15     expertise in the document management system and make     15       that --
  16     sure we can take whatever benefit that can be obtained   16     MR JAY: Yes.
@@ -234,14 +234,14 @@
 (+44) 207 404 1400                                                                                                      London EC4A 2DY
                                                         Leveson Inquiry Initial Hearing                                4 October 2011
 
-  1   LORD JUSTICE LEVESON: Therefore, I am required by the terms 1           solidify until we have the complete picture.
+  1   LORD JUSTICE LEVESON: Therefore, I am required by the terms  1          solidify until we have the complete picture.
   2     of reference, which are far more detailed than in other    2        LORD JUSTICE LEVESON: Where are we on responses to the
   3     inquiries, to look at some specific issues now, before     3          notices for evidence and the requests for evidence,
   4     the investigation can go into the sort of detail that      4          because there are two things.
   5     would be necessary, which would have to await the          5        MR JAY: It would be right to say that all recipients, on my
   6     conclusion of any police investigation.                    6          understanding of Section 21 notices, have engaged with
   7   MR JAY: Yes.                                                 7          the Inquiry. Some have asked for an extension of time,
-  8   LORD JUSTICE LEVESON: To try to go into further explanation 8           which have been granted. There are no specific matters
+  8   LORD JUSTICE LEVESON: To try to go into further explanation  8          which have been granted. There are no specific matters
   9     seems to me to be rather difficult and, to some extent,    9          which cause us concern and which we need to draw to your
  10     I have to wait and see what evidence is generated and     10          attention at this stage.
  11     what I can get to grips with and how far I can go         11        LORD JUSTICE LEVESON: Okay.
@@ -261,15 +261,15 @@
  25     and, if so, how much notice? Well, the answer is: we      25          working to do so within extended times and I am very
                                 Page 17                                                           Page 19
 
- 1      will give as much notice as we can. It goes without       1           keen to receive things as and when they are ready,
- 2      saying we won't be telling witnesses the night before     2           rather than have it all bundled up in one great pile if
- 3      that they are on the stand, as it were, the following     3           that is not inconvenient.
- 4      morning.                                                  4         MR JAY: Yes. Unless I hear otherwise, during the course of
- 5    LORD JUSTICE LEVESON: I don't think we have stands in this  5           the morning, on that specific matter, namely compliance
- 6      country, Mr Jay.                                          6           with the Section 21 notices, there is nothing I need to
- 7    MR JAY: That is true.                                       7           bring specifically to your attention. It is all being
- 8    LORD JUSTICE LEVESON: Nobody should draw inferences about 8             dealt with behind the scenes and on an amicable basis,
- 9      the way in which we will try to organise the time from    9           but if that is wrong I will be told.
+  1     will give as much notice as we can. It goes without       1           keen to receive things as and when they are ready,
+  2     saying we won't be telling witnesses the night before     2           rather than have it all bundled up in one great pile if
+  3     that they are on the stand, as it were, the following     3           that is not inconvenient.
+  4     morning.                                                  4         MR JAY: Yes. Unless I hear otherwise, during the course of
+  5   LORD JUSTICE LEVESON: I don't think we have stands in this  5           the morning, on that specific matter, namely compliance
+  6     country, Mr Jay.                                          6           with the Section 21 notices, there is nothing I need to
+  7   MR JAY: That is true.                                       7           bring specifically to your attention. It is all being
+  8   LORD JUSTICE LEVESON: Nobody should draw inferences about   8           dealt with behind the scenes and on an amicable basis,
+  9     the way in which we will try to organise the time from    9           but if that is wrong I will be told.
  10     the way we have organised these hearings. The fact is    10         LORD JUSTICE LEVESON: I hope if it ceases to be amicable
  11     that I will want to set out a broad way forward with     11           I will be told.
  12     some identification of the time available, not least so  12         MR JAY: Yes.
@@ -321,7 +321,7 @@
 
   1     subject to Rule 10.1.                                     1     section 24 of the Act. I put forward some suggestions
   2   MR JAY: Yes.                                                2     based on what was determined in another enquiry, the
-  3   LORD JUSTICE LEVESON: Subject to that provision, it makes 3       Baha Mousa Inquiry, and the ruling of Sir William Gage
+  3   LORD JUSTICE LEVESON: Subject to that provision, it makes   3     Baha Mousa Inquiry, and the ruling of Sir William Gage
   4     it clear that only counsel to the Inquiry and the         4     is available on his website, the matter is subject to
   5     Inquiry panel, which in that case is me and me alone,     5     detailed submissions and then a detailed ruling. We
   6     may ask questions of that witness.                        6     have made some suggestions really on the back of that,
@@ -417,7 +417,7 @@
   7   LORD JUSTICE LEVESON: Absolutely.                           7        start and don't stop. In other words, we keep going so
   8   MR DAVIES: Nonetheless, there has to come a point at which  8        that we would move straight from opening submissions
   9     we say, "This is the way we think it should go".          9        into evidence.
- 10   LORD JUSTICE LEVESON: Absolutely, and I am happy for that 10       MR DAVIES: Yes.
+ 10   LORD JUSTICE LEVESON: Absolutely, and I am happy for that  10      MR DAVIES: Yes.
  11     whenever you wish. Certainly, it will be helpful to      11      LORD JUSTICE LEVESON: Now, that is not a timeframe that you
  12     have it set out at the beginning, not least because one  12        have been told before because, to some extent, it has
  13     can then examine the evidence against the possible       13        been one that I don't say has been forced upon me
@@ -478,14 +478,14 @@
  10     possibility because, otherwise, the plan of action will  10   what happened when. Then there is obviously a great
  11     just lose its momentum.                                  11   deal of territory in between. As a party who is
  12   MR DAVIES: Yes.                                            12   necessarily quite closely engaged in some of the history
- 13   LORD JUSTICE LEVESON: Would that cause your clients any 13      it would help us to know, if at all possible, at what
+ 13   LORD JUSTICE LEVESON: Would that cause your clients any    13   it would help us to know, if at all possible, at what
  14     difficulty?                                              14   level the Inquiry intends to pitch its enquiry into the
  15   MR DAVIES: It will cause, I am sure, the obvious           15   narrative.
  16     difficulties that busy people would prefer to prepare    16 LORD JUSTICE LEVESON: Your question contains within it the
  17     things once and do them once but those are probably not  17   answer to your question. It is impossible to identify
  18     difficulties which weigh very heavily in the scales as   18   the level of detail and certainly impossible until
  19     against the importance of the Inquiry.                   19   I have had submissions from the Director of Public
- 20   LORD JUSTICE LEVESON: Yes, but the preparation would be 20      Prosecutions as to the extent to which he is concerned
+ 20   LORD JUSTICE LEVESON: Yes, but the preparation would be    20   Prosecutions as to the extent to which he is concerned
  21     different, wouldn't it?                                  21   that what I might do could prejudice proceedings.
  22   MR DAVIES: It will.                                        22   I think I identified the need for that very early on.
  23   LORD JUSTICE LEVESON: I take the point and, indeed, people 23      All I can say is that I am mindful of the issue, but
@@ -524,15 +524,15 @@
 (+44) 207 404 1400                                                                                          London EC4A 2DY
                                                            Leveson Inquiry Initial Hearing                                           4 October 2011
 
- 1    LORD JUSTICE LEVESON: As much notice that can be given will 1               five minutes sufficient -- just to provide a break, and
- 2      be given.                                                 2               I have taken you quite slowly because a number of the
- 3    MR DAVIES: Yes.                                                       3     issues are raised by a number of the others, so
- 4    LORD JUSTICE LEVESON: You can rest assured that my                    4     hopefully, the answers that we have discussed, that we
- 5      expertise may not extend to all aspects of the work of              5     have analysed, help some of them too. We will just take
- 6      the press but it certainly does extend to the pressures             6     five minutes.
- 7      of giving evidence in any forum and I well understand               7   (11.35 am)
- 8      the difficulties that even the most robust might face.              8                  (A short break)
- 9      So such assistance as can be provided to allow witnesses            9   (11.40 am)
+  1   LORD JUSTICE LEVESON: As much notice that can be given will           1     five minutes sufficient -- just to provide a break, and
+  2     be given.                                                           2     I have taken you quite slowly because a number of the
+  3   MR DAVIES: Yes.                                                       3     issues are raised by a number of the others, so
+  4   LORD JUSTICE LEVESON: You can rest assured that my                    4     hopefully, the answers that we have discussed, that we
+  5     expertise may not extend to all aspects of the work of              5     have analysed, help some of them too. We will just take
+  6     the press but it certainly does extend to the pressures             6     five minutes.
+  7     of giving evidence in any forum and I well understand               7   (11.35 am)
+  8     the difficulties that even the most robust might face.              8                  (A short break)
+  9     So such assistance as can be provided to allow witnesses            9   (11.40 am)
  10     to prepare appropriately will be given, but there are              10 MR DAVIES: You will be pleased to know I don't have
  11     limits to that, obviously.                                         11     anything to add and there is nothing that I can sensibly
  12 MR DAVIES: Then the last point is really the reverse side              12     say now about the date. We will have to discuss it with
@@ -610,15 +610,15 @@
  25   areas.                                                                  25 MR DINGEMANS: -- but the proposition is that it is helpful
                              Page 41                                                                          Page 43
 
- 1    LORD JUSTICE LEVESON: That is a different point entirely.    1                 to identify, for example, those specific matters which
- 2      I hope that all those involved in this Inquiry will        2                 have occurred to the Inquiry, which has access to
- 3      liaise very closely with counsel to the Inquiry to         3                 materials that the core participants don't, because it
- 4      ensure that this particular vessel proceeds in             4                 is collating all the material, it will then share it,
- 5      an orderly fashion. I anticipate that there may be real    5                 and in those circumstances, the collaboration that you
- 6      value in meetings outside the formal hearings of the       6                 ask for is likely to be more readily available.
- 7      Inquiry between counsel to try to identify the best ways   7               LORD JUSTICE LEVESON: I quite agree. We will hear what
- 8      forward in specific areas and I take the view, I hope      8                 Mr Jay has to say about that at the end but my immediate
- 9      accurately, that everybody sees the value in getting       9                 reaction is that that is exactly the sort of thing that
+  1   LORD JUSTICE LEVESON: That is a different point entirely.    1                 to identify, for example, those specific matters which
+  2     I hope that all those involved in this Inquiry will        2                 have occurred to the Inquiry, which has access to
+  3     liaise very closely with counsel to the Inquiry to         3                 materials that the core participants don't, because it
+  4     ensure that this particular vessel proceeds in             4                 is collating all the material, it will then share it,
+  5     an orderly fashion. I anticipate that there may be real    5                 and in those circumstances, the collaboration that you
+  6     value in meetings outside the formal hearings of the       6                 ask for is likely to be more readily available.
+  7     Inquiry between counsel to try to identify the best ways   7               LORD JUSTICE LEVESON: I quite agree. We will hear what
+  8     forward in specific areas and I take the view, I hope      8                 Mr Jay has to say about that at the end but my immediate
+  9     accurately, that everybody sees the value in getting       9                 reaction is that that is exactly the sort of thing that
  10     a result from this Inquiry which moves this debate on     10                 I would expect to happen without it being a formal
  11     and, therefore, that everybody will work to try to find   11                 identification of a list which then everybody has to
  12     that solution and, therefore, the more that can be done   12                 comment on and grows barnacles.
@@ -658,25 +658,25 @@
  15     dealing with.                                            15      represent others, but to call a person who thinks it may
  16   LORD JUSTICE LEVESON: I hope nobody is going to be         16      be the witness statement only, even telling them that is
  17     terrified by the experience.                             17      likely to provide massive reassurance. Then to say,
- 18   MR DINGEMANS: Sir, you will have seen enough witnesses in 18       "And also topics, for example, doorstepping", if that
+ 18   MR DINGEMANS: Sir, you will have seen enough witnesses in  18      "And also topics, for example, doorstepping", if that
  19     your time to realise some enjoy it, some play to the     19      has been identified as a topic or something else, we
  20     gallery and some are terrified by the experience.        20      respectfully submit is so obvious that it almost goes
- 21   LORD JUSTICE LEVESON: Yes, I am not suggesting many will 21        without saying, which is why this is my second point.
+ 21   LORD JUSTICE LEVESON: Yes, I am not suggesting many will   21      without saying, which is why this is my second point.
  22     fall into the first category, I will stop anybody        22    LORD JUSTICE LEVESON: Why doesn't it appear in the Act?
  23     falling into the second category. Yes, I suppose it is   23    MR DINGEMANS: Because the Act says "fairness". The Act
  24     a question of the granularity of the information.        24      says "fairness" and the proposition that you would call
  25   MR DINGEMANS: If there is to be none and, sir, you are to  25      a person to be subjected to public questioning, without
                             Page 45                                                                Page 47
 
-  1   rule that we are to have none, we respectfully submit       1      any notice, we respectfully submit would infringe all
-  2   that would infringe all of the principles of the Act and    2      the principles of --
-  3   it is plainly not your intention to be unfair and, of       3    LORD JUSTICE LEVESON: It happens all the time in crime.
-  4   course, no one is trying to add to the immense burdens      4    MR DINGEMANS: Even in crime people have at least some idea
-  5   on my learned friend, but to put a witness up who has       5      of the broad areas.
-  6   provided a statement dated 16 September at some stage in    6    LORD JUSTICE LEVESON: They have made a witness statement
-  7   the future, with no idea about the areas that they are      7      and they are cross-examined on a wide range of issues
-  8   going to be asked, what might occur overnight in the        8      and, indeed, I was personally involved in an Inquiry,
-  9   bath or might have been suggested informally by someone     9      acting for a witness in that case, in which the word
+  1   rule that we are to have none, we respectfully submit        1     any notice, we respectfully submit would infringe all
+  2   that would infringe all of the principles of the Act and     2     the principles of --
+  3   it is plainly not your intention to be unfair and, of        3   LORD JUSTICE LEVESON: It happens all the time in crime.
+  4   course, no one is trying to add to the immense burdens       4   MR DINGEMANS: Even in crime people have at least some idea
+  5   on my learned friend, but to put a witness up who has        5     of the broad areas.
+  6   provided a statement dated 16 September at some stage in     6   LORD JUSTICE LEVESON: They have made a witness statement
+  7   the future, with no idea about the areas that they are       7     and they are cross-examined on a wide range of issues
+  8   going to be asked, what might occur overnight in the         8     and, indeed, I was personally involved in an Inquiry,
+  9   bath or might have been suggested informally by someone      9     acting for a witness in that case, in which the word
  10   else, we respectfully suggest would be plainly wrong.       10     I used to describe what had happened to him was
  11 LORD JUSTICE LEVESON: It depends on the level of detail       11     "hijacked".
  12   that one is talking about. If somebody has made a very,     12 MR DINGEMANS: The potential unfairness of that has been
@@ -699,7 +699,7 @@
 (+44) 207 404 1400                                                                                                    London EC4A 2DY
                                                  Leveson Inquiry Initial Hearing                                   4 October 2011
 
-  1   LORD JUSTICE LEVESON: Splendid. Right. We have a very 1             grateful for the steer that your Lordship will give as
+  1   LORD JUSTICE LEVESON: Splendid. Right. We have a very        1      grateful for the steer that your Lordship will give as
   2     interesting submission or representative who -- no,        2      much notice as possible on that regard.
   3     Mr Glen must be next. He is the next core participant,     3         But in terms of lines of questioning and indications
   4     Mr Glen.                                                   4      of documents, again, I don't think what we were
@@ -709,7 +709,7 @@
   8     have fallen by the wayside. The standard of proof          8      we would submit that this is an area where the witness
   9     point, I think our concern, at this stage, was this,       9      should be provided with at least the outline areas or
  10     that it was potentially premature.                        10      subject or issues which he or she is likely to face
- 11   LORD JUSTICE LEVESON: I am very comfortable to defer that. 11       questioning on.
+ 11   LORD JUSTICE LEVESON: I am very comfortable to defer that.  11      questioning on.
  12     What I wanted to do was to put on the table some of the   12         Similarly, in relation to documents, particularly
  13     issues that are involved, right?                          13      given in this case, as I understand it, the witness will
  14   MR GLEN: I am grateful. It may be that the civil standard   14      be -- the document concerned will be flashed up on
@@ -822,14 +822,14 @@
   5     to have one's point of view heard by the Inquiry --       5   an opportunity of commenting on it without having to
   6   LORD JUSTICE LEVESON: Yes. That is right.                   6   become a core participant.
   7   MR VINALL: -- which is -- as I am demonstrating now.        7       The reason we are different from the various
-  8   LORD JUSTICE LEVESON: I am not sure how far, but to have 8      examples that you have given, sir, is that, plainly, the
+  8   LORD JUSTICE LEVESON: I am not sure how far, but to have    8   examples that you have given, sir, is that, plainly, the
   9     your point of view in the ultimate issues, of course,     9   interest of Trinity Mirror doesn't need any further
  10     that's why I say making submissions at the end in        10   elaboration. It goes beyond that. We have an interest
  11     writing would be fine, but the level of participation is 11   not only in the outcome of the enquiry but in the
  12     a slightly different question.                           12   Inquiry's processes.
  13   MR VINALL: Sir, you asked me how I put it.                 13 LORD JUSTICE LEVESON: Then all you are saying, Mr Vinall,
  14   LORD JUSTICE LEVESON: I do, yes I will let you.            14   is very clear. What you are saying is, "We actually
- 15   MR VINALL: As both Mr Jay and you have already noted this 15    qualify to be core participants and that makes us
+ 15   MR VINALL: As both Mr Jay and you have already noted this  15   qualify to be core participants and that makes us
  16     morning, the overriding criteria for this Inquiry is     16   different." Well, if you do, do it.
  17     fairness and it is my submission that there are certain  17 MR VINALL: Sir, it is not so much that we qualify to be
  18     classes of documents, clearly not every document that    18   core participants, that will be a matter for you, sir,
@@ -837,7 +837,7 @@
  20     documents where, in my submission, fairness to my client 20   Section 21 notices requiring documents to be provided by
  21     does require that it should have notice of what is being 21   Trinity Mirror. So when there is discussion about the
  22     said by the core participants and by the Inquiry team.   22   protocol for how documents of that kind are to be
- 23   LORD JUSTICE LEVESON: I am not so sure about that because 23    handled, then the interest -- they are our documents
+ 23   LORD JUSTICE LEVESON: I am not so sure about that because  23   handled, then the interest -- they are our documents
  24     you could have applied to be core participants. Now,     24   too.
  25     you didn't and that's fair enough, because core          25 LORD JUSTICE LEVESON: But there are 130-odd people who have
                             Page 57                                                         Page 59
@@ -873,15 +873,15 @@
 (+44) 207 404 1400                                                                                          London EC4A 2DY
                                                  Leveson Inquiry Initial Hearing                                              4 October 2011
 
-  1     parties, which do affect Trinity Mirror's interests and,    1      which has then fed into the submissions that Mr Jay has
-  2     in my submission, it is appropriate that Trinity Mirror     2      made to you today. Everybody else in the room knows
-  3     should see those submissions where they affect its          3      what those submissions have been and what that
-  4     interest.                                                   4      dialogue --
-  5         Sir, if you accept that, there is then a decision to    5    LORD JUSTICE LEVESON: So you want to see the submissions by
-  6     be made as to whether one considers what fairness           6      other core participants?
-  7     requires, case by case, document by document, or whether    7    MR VINALL: Yes, sir, certainly we do.
-  8     it is not easier for everybody, if you simply add my        8    LORD JUSTICE LEVESON: They may have something to say about
-  9     clients to the list of those parties who are to receive     9      that.
+  1     parties, which do affect Trinity Mirror's interests and,     1     which has then fed into the submissions that Mr Jay has
+  2     in my submission, it is appropriate that Trinity Mirror      2     made to you today. Everybody else in the room knows
+  3     should see those submissions where they affect its           3     what those submissions have been and what that
+  4     interest.                                                    4     dialogue --
+  5         Sir, if you accept that, there is then a decision to     5   LORD JUSTICE LEVESON: So you want to see the submissions by
+  6     be made as to whether one considers what fairness            6     other core participants?
+  7     requires, case by case, document by document, or whether     7   MR VINALL: Yes, sir, certainly we do.
+  8     it is not easier for everybody, if you simply add my         8   LORD JUSTICE LEVESON: They may have something to say about
+  9     clients to the list of those parties who are to receive      9     that.
  10     the documents, which are provided to core participants,     10 MR VINALL: Perhaps my understanding was, sir, that those
  11     so that where there is an issue -- and it may not arise     11     submissions were being disclosed to other core
  12     very often -- where there is an issue where                 12     participants.
@@ -990,15 +990,15 @@
 (+44) 207 404 1400                                                                                           London EC4A 2DY
                                                       Leveson Inquiry Initial Hearing                                 4 October 2011
 
- 1      proposed approach towards standard of proof may be one     1         the rules and you may make the point, and I haven't
- 2      of those. I am not going to say anything more about        2         thought about it, that, absent your being present or
- 3      that at the moment, conscious as I am that the topic is    3         somebody on behalf of your client being present on every
- 4      going to be deferred. Equally, the question of putting     4         day, how will you know if the witness gives evidence
- 5      in the report concern arising out of a suspicion is        5         directly relating to your client?
- 6      another topic upon which we are likely to have             6       MR CHAWLA: The answer, actually, is simple: we wouldn't
- 7      submissions.                                               7         know. We wouldn't know until we saw the transcript
- 8         But equally, there are other areas upon which she       8         unless we were here every day, unless, of course, we had
- 9      has no special interest or particular contribution to      9         advance notice of issues that may touch upon that. We
+  1     proposed approach towards standard of proof may be one     1         the rules and you may make the point, and I haven't
+  2     of those. I am not going to say anything more about        2         thought about it, that, absent your being present or
+  3     that at the moment, conscious as I am that the topic is    3         somebody on behalf of your client being present on every
+  4     going to be deferred. Equally, the question of putting     4         day, how will you know if the witness gives evidence
+  5     in the report concern arising out of a suspicion is        5         directly relating to your client?
+  6     another topic upon which we are likely to have             6       MR CHAWLA: The answer, actually, is simple: we wouldn't
+  7     submissions.                                               7         know. We wouldn't know until we saw the transcript
+  8        But equally, there are other areas upon which she       8         unless we were here every day, unless, of course, we had
+  9     has no special interest or particular contribution to      9         advance notice of issues that may touch upon that. We
  10     make, such as -- broadcasting of these proceedings or     10         have no --
  11     public funding, being two examples on which she has no    11       LORD JUSTICE LEVESON: The problem is: we won't necessarily
  12     representations to make, with the result that our         12         know, will we?
@@ -1133,15 +1133,15 @@
  25   applications was at the end of module 1. You will see              25         "The chairman will decide which witnesses will be
                                Page 77                                                              Page 79
 
- 1      already how that would cause very considerable            1             called to give oral evidence and which evidence may be
- 2      difficulties.                                             2             read out", et cetera.
- 3    LORD JUSTICE LEVESON: No, I won't embark upon any aspect of 3                18.2 says:
- 4      module 2 without making sure that those who may be core   4                "Counsel to the Inquiry will use their judgment on
- 5      participants in module 2 have had ample opportunity: (a)  5             a case-by-case basis as to whether a witness will be
- 6      to make applications; and (b) to participate effectively  6             required to give oral evidence to the Inquiry on all or
- 7      in that part of the Inquiry. That much, I would hope,     7             some of the matters covered in a witness statement. All
- 8      would be self-evident.                                    8             of the part of the statement will be taken as read or
- 9    MR PHILLIPS: Yes. Sir, that's the concern. As I say, in     9             some or all of the statement will be summarised."
+  1     already how that would cause very considerable            1             called to give oral evidence and which evidence may be
+  2     difficulties.                                             2             read out", et cetera.
+  3   LORD JUSTICE LEVESON: No, I won't embark upon any aspect of 3                18.2 says:
+  4     module 2 without making sure that those who may be core   4                "Counsel to the Inquiry will use their judgment on
+  5     participants in module 2 have had ample opportunity: (a)  5             a case-by-case basis as to whether a witness will be
+  6     to make applications; and (b) to participate effectively  6             required to give oral evidence to the Inquiry on all or
+  7     in that part of the Inquiry. That much, I would hope,     7             some of the matters covered in a witness statement. All
+  8     would be self-evident.                                    8             of the part of the statement will be taken as read or
+  9   MR PHILLIPS: Yes. Sir, that's the concern. As I say, in     9             some or all of the statement will be summarised."
  10     the intervening period, however long it is, our concern  10                To us, there seemed to be an inconsistency there
  11     is to ensure that anything that might affect our         11             between the chairman having the decision on 15 and
  12     interests is shared with us, so that we can make our own 12             counsel to the Inquiry having the judgment decision or
@@ -1185,7 +1185,7 @@
  19     but I can see that that is not what you want. All I am    19     considerable concern to us. It may be, however, that
  20     then saying is, if it is to be later, then let it not be  20     you broaden the circle to include some of my learned
  21     too late, ie so as to prejudice --                        21     friends, who anticipate receiving material on a basis
- 22   LORD JUSTICE LEVESON: I take that point and it may be that 22      other than the fact that they are core participants.
+ 22   LORD JUSTICE LEVESON: I take that point and it may be that  22     other than the fact that they are core participants.
  23     there is no harm to be done in getting on with it. But    23   LORD JUSTICE LEVESON: Yes, the question is whether I ought
  24     I will think about that.                                  24     to create a new category of, one could call them --
  25   MR PHILLIPS: Thank you.                                     25     I know the word is used in the Act, but in a different
@@ -1223,25 +1223,25 @@
                                                 Leveson Inquiry Initial Hearing                                               4 October 2011
 
   1     participants, but if it is to be something wider than --   1   wish to make witness statements should have the
-  2   LORD JUSTICE LEVESON: No, I agree with you. That is why 2        opportunity to but there is no reason why, in discussion
+  2   LORD JUSTICE LEVESON: No, I agree with you. That is why      2   opportunity to but there is no reason why, in discussion
   3     I am concerned about the width and the breadth of what     3   with the team, you shouldn't focus on a number of
   4     Mr Vinall has suggested.                                   4   specific witnesses who illustrate the different problems
   5   MR GARNHAM: Yes, all I submit is it needs to be thought      5   and allow us to start with, perhaps, a comparatively
   6     through carefully and we would want to make submissions    6   small number and, if others then want at some later
   7     on it.                                                     7   stage to come into the picture and it is appropriate
-  8   LORD JUSTICE LEVESON: Yes, in my effort to want to make 8        that they give oral evidence, as opposed merely to
+  8   LORD JUSTICE LEVESON: Yes, in my effort to want to make      8   that they give oral evidence, as opposed merely to
   9     sure that I heard all those who were interested in those   9   having their statements put in, then that can be
  10     areas where they were interested, I may have created      10   organised as well. You don't have to have proved every
  11     an issue, which I just have to grapple with and say --    11   single one of your clients before we can press the
  12   MR GARNHAM: You may say no, sir, but if you are             12   button.
  13     contemplating saying yes, then we may have submissions.   13 MR SHERBORNE: No, sir, we anticipated that and we have
- 14   LORD JUSTICE LEVESON: All right, I have it, thank you very 14    spoken to the Inquiry team. We are putting together
+ 14   LORD JUSTICE LEVESON: All right, I have it, thank you very  14   spoken to the Inquiry team. We are putting together
  15     much, Mr Garnham. I think we have come to Mr Sherborne.   15   a number of individuals. I think a comparatively
  16             Submissions by MR SHERBORNE                       16   smaller number, certainly, than 46 but, of course, that
  17   MR SHERBORNE: I am very conscious of the time and I am also 17   does take time and, therefore, what I would urge at this
  18     conscious of the fact that Mr Jay needs to speak after    18   stage is that, if a provisional date is going to be
  19     I do. So I will keep this very brief, if I may.           19   fixed, at least, as I say, in provisional terms, it is
- 20   LORD JUSTICE LEVESON: If I go over, I will go over. Do you 20    one that is later on in November rather than earlier.
+ 20   LORD JUSTICE LEVESON: If I go over, I will go over. Do you  20   one that is later on in November rather than earlier.
  21     know that has happened to me before?                      21 LORD JUSTICE LEVESON: So you won't be impressed to know
  22   MR SHERBORNE: I am sure it has, sir, I am sure it has       22   that my original anxiety was to press for slightly
  23     happened to me as well. If I can help you, I will try.    23   earlier?
@@ -1249,15 +1249,15 @@
  25     put in.                                                   25   me and who represents the victims with a certain amount
                             Page 85                                                               Page 87
 
-  1 LORD JUSTICE LEVESON: Yes.                                   1      of fear.
-  2 MR SHERBORNE: Can I take you very briefly through a few      2    LORD JUSTICE LEVESON: Only because you know as much and as
-  3   points we have made? There is the familiar cri de          3      well as everybody else the territory that has to be
-  4   coeur, as regards warning. I am not going to repeat        4      travelled before next summer.
-  5   that, we have all dealt with that this morning. But        5        I am not going to be overly constrained but I am
-  6   coming specifically, if I may, to the start date for the   6      very keen to keep the focus because I am conscious that
-  7   oral hearing, I am not seeking to pin you down, but I am   7      whatever I come up with is likely to generate a debate,
-  8   seeking to deter you from entering the tunnel that you     8      a debate among the media who may or may not be polite,
-  9   referred to, quite as quickly as may be envisaged,         9      a debate among the political groupings, and
+  1 LORD JUSTICE LEVESON: Yes.                                    1     of fear.
+  2 MR SHERBORNE: Can I take you very briefly through a few       2   LORD JUSTICE LEVESON: Only because you know as much and as
+  3   points we have made? There is the familiar cri de           3     well as everybody else the territory that has to be
+  4   coeur, as regards warning. I am not going to repeat         4     travelled before next summer.
+  5   that, we have all dealt with that this morning. But         5       I am not going to be overly constrained but I am
+  6   coming specifically, if I may, to the start date for the    6     very keen to keep the focus because I am conscious that
+  7   oral hearing, I am not seeking to pin you down, but I am    7     whatever I come up with is likely to generate a debate,
+  8   seeking to deter you from entering the tunnel that you      8     a debate among the media who may or may not be polite,
+  9   referred to, quite as quickly as may be envisaged,          9     a debate among the political groupings, and
  10   because once we are in, I agree, we are in until we get    10     a reconsideration of the way, perhaps, I don't know, in
  11   to the glorious light at the other end.                    11     which regulation, self regulation, whatever comes out,
  12      Now, in terms of specific dates, perhaps like other     12     is organised, which everybody is going to want to get on
@@ -1307,15 +1307,15 @@
  25  my clients are concerned, in being prepared. Because           25     the Inquiry team is very clear on that.
                              Page 89                                                                Page 91
 
-  1     once we do start and I presume, without necessarily         1    LORD JUSTICE LEVESON: I am not --
-  2     knowledge, that it will be the core participant victims     2    MR SHERBORNE: And the protocol certainly reflects that.
-  3     who will be giving evidence first.                          3    LORD JUSTICE LEVESON: Good.
-  4   LORD JUSTICE LEVESON: That would be my wish, because          4    MR SHERBORNE: Then finally, I think it is simply the
-  5     I think it is probably the right way to start.              5      question which goes beyond anything that is in Mr Jay's
-  6   MR SHERBORNE: Of course.                                      6      note and that is, as you will have seen from my note,
-  7   LORD JUSTICE LEVESON: But I am not suggesting that, if they   7      that there are two further individuals who wish to apply
-  8     don't, they have missed the bus, another one will come      8      for core participant status. I don't know whether you
-  9     along, but I am worried about time.                         9      wish me to deal with that before the adjournment or
+  1     once we do start and I presume, without necessarily          1   LORD JUSTICE LEVESON: I am not --
+  2     knowledge, that it will be the core participant victims      2   MR SHERBORNE: And the protocol certainly reflects that.
+  3     who will be giving evidence first.                           3   LORD JUSTICE LEVESON: Good.
+  4   LORD JUSTICE LEVESON: That would be my wish, because           4   MR SHERBORNE: Then finally, I think it is simply the
+  5     I think it is probably the right way to start.               5     question which goes beyond anything that is in Mr Jay's
+  6   MR SHERBORNE: Of course.                                       6     note and that is, as you will have seen from my note,
+  7   LORD JUSTICE LEVESON: But I am not suggesting that, if they    7     that there are two further individuals who wish to apply
+  8     don't, they have missed the bus, another one will come       8     for core participant status. I don't know whether you
+  9     along, but I am worried about time.                          9     wish me to deal with that before the adjournment or
  10   MR SHERBORNE: I understand that. I understand that. Shall     10     whether you would rather hear from Mr Jay in response to
  11     I move on then very --                                      11     everyone's submissions and that I come back once again
  12   LORD JUSTICE LEVESON: I would have thought the opening        12     and deal with the point on my own and perhaps with very
@@ -1362,7 +1362,7 @@
  22         So our position is that, unless we specifically ask   22     can't research it, you can't think about it, you have to
  23      for confidentiality for something, we don't mind who it  23     answer now", because, even if one did say that, and I am
  24      goes to. We regard it as public.                         24     not suggesting that I would, the answer becomes
- 25   LORD JUSTICE LEVESON: That is very helpful. Thank you very 25      valueless if the evidence is later undermined by
+ 25   LORD JUSTICE LEVESON: That is very helpful. Thank you very  25     valueless if the evidence is later undermined by
                             Page 93                                                          Page 95
 
   1  much. Does anybody else want to say anything on that         1      research that does take place on the document.
@@ -1399,7 +1399,7 @@
   1     witness, would you like to deal with this topic please?   1        MR JAY: Yes, that is right. So the Inquiry then will be
   2   MR JAY: Yes, notice of that should be given. That is what   2          giving advance notice -- how much advance notice is
   3     the rules require.                                        3          going to depend, but as much as possible is all I can
-  4   LORD JUSTICE LEVESON: But you do that, hopefully, in good 4            say at this stage -- be giving advance notice on topics
+  4   LORD JUSTICE LEVESON: But you do that, hopefully, in good   4          say at this stage -- be giving advance notice on topics
   5     time.                                                     5          and areas to be covered by a witness in oral evidence
   6   MR JAY: Yes. Matters -- topics or areas which are already   6          arising out of his or her witness statement. If that
   7     covered in a witness statement, and after all, the model  7          advance notice is also given to the other core
@@ -1423,15 +1423,15 @@
  25     asked someone to deal, and I take Mr Chawla's example,   25          to us the, as it were, the subject matters which should
                                  Page 97                                                         Page 99
 
- 1      19 topics, and 15 of them, we are content, simply form     1         be covered because then that will start the ball
- 2      part of the evidence to the Inquiry --                     2         rolling.
- 3    MR JAY: Yes.                                                 3       LORD JUSTICE LEVESON: Yes.
- 4    LORD JUSTICE LEVESON: -- they may or may not be contentious 4        MR JAY: We won't necessarily come back to them immediately
- 5      in any way but they are not features upon which the        5         but it will ensure that in good time the relevant ground
- 6      Inquiry want to focus and "focus" is the watch word for    6         is covered.
- 7      all the reasons that I have been talking about.            7       LORD JUSTICE LEVESON: My concern is that, whether I accept
- 8    MR JAY: Yes.                                                 8         the language used, I think it was by Mr Dingemans, about
- 9    LORD JUSTICE LEVESON: Then we get the best value out of the 9          witnesses or not, that for some the giving of evidence
+  1     19 topics, and 15 of them, we are content, simply form     1         be covered because then that will start the ball
+  2     part of the evidence to the Inquiry --                     2         rolling.
+  3   MR JAY: Yes.                                                 3       LORD JUSTICE LEVESON: Yes.
+  4   LORD JUSTICE LEVESON: -- they may or may not be contentious  4       MR JAY: We won't necessarily come back to them immediately
+  5     in any way but they are not features upon which the        5         but it will ensure that in good time the relevant ground
+  6     Inquiry want to focus and "focus" is the watch word for    6         is covered.
+  7     all the reasons that I have been talking about.            7       LORD JUSTICE LEVESON: My concern is that, whether I accept
+  8   MR JAY: Yes.                                                 8         the language used, I think it was by Mr Dingemans, about
+  9   LORD JUSTICE LEVESON: Then we get the best value out of the  9         witnesses or not, that for some the giving of evidence
  10     witness if we say, actually, the specific topics that we  10         is, indeed, a difficult exercise and I will want to make
  11     want to ask are going to be topics 13, 15 and 17.         11         that exercise as easy an experience as possible on the
  12   MR JAY: Yes.                                                12         basis that this isn't a trial. I am simply looking at
@@ -1444,7 +1444,7 @@
  19     statement, but that, as far as the tribunal, the Inquiry  19         introduce the witness through his or her own counsel --
  20     is concerned, we would be focusing on that.               20       MR JAY: Yes.
  21   MR JAY: Yes.                                                21       LORD JUSTICE LEVESON: -- simply to get used to talking in
- 22   LORD JUSTICE LEVESON: If you have learnt that, because you 22          court and answering questions. So I wouldn't have
+ 22   LORD JUSTICE LEVESON: If you have learnt that, because you  22         court and answering questions. So I wouldn't have
  23     have been asked that, actually, we would like to ask      23         a problem about that. It is not a request to make for
  24     questions on topic 16 as well, well that can be           24         every single witness but I can well understand there
  25     transmitted in addition.                                  25         could be some witnesses for whom that would be
@@ -1457,11 +1457,11 @@
   1     an important way of trying to allow the witness to relax   1       That is the advance notice point, and it may be the
   2     to such extent, if possible.                               2     best way forward is for me to finalise the note which is
   3   MR JAY: Yes.                                                 3     still, I suppose, in draft form, to incorporate the
-  4   LORD JUSTICE LEVESON: I am not unmindful of the pressures 4        advance notice points so that they are made explicit and
+  4   LORD JUSTICE LEVESON: I am not unmindful of the pressures    4     advance notice points so that they are made explicit and
   5     of giving evidence.                                        5     then a final version of the note, which was provided to
   6   MR JAY: Yes. It is the role of counsel to the Inquiry, to    6     the parties at the beginning of last week can then be
   7     the best it can, not to unsettle witnesses.                7     placed on the website. The same will apply, of course,
-  8   LORD JUSTICE LEVESON: I wasn't for a moment suggesting that 8      to the protocol. They will cease to be draft protocols.
+  8   LORD JUSTICE LEVESON: I wasn't for a moment suggesting that  8     to the protocol. They will cease to be draft protocols.
   9     you would approach it differently, but with the best       9   LORD JUSTICE LEVESON: I will ask everybody about that in
  10     will in the world, you will not have quite the same       10     a moment. I think that might be the way forward.
  11     relationship with the witnesses --                        11   MR JAY: It will be understood that the note and the
@@ -1513,7 +1513,7 @@
                                                 Leveson Inquiry Initial Hearing                                               4 October 2011
 
   1   MR JAY: Of the risk, yes.                                   1    LORD JUSTICE LEVESON: All right.
-  2   LORD JUSTICE LEVESON: The point being that that risk does 2      MR JAY: Finally, can I endorse from our perspective a point
+  2   LORD JUSTICE LEVESON: The point being that that risk does   2    MR JAY: Finally, can I endorse from our perspective a point
   3     not arise if, for example, there is generic evidence of   3      you made before lunch, that it already appears to us
   4     what was happening in a particular news room of which     4      that this Inquiry could take, in an ideal world in which
   5     Mrs Brooks was the editor, unless it impacts on her, but  5      we do not live, three or four years. That is not what
@@ -1523,7 +1523,7 @@
   9     something that is specific about which her legal          9      within about a year. The difference between a one year
  10     representatives may have a right to apply under 10.3.    10      and three year Inquiry, in terms of contents of your
  11   MR JAY: Yes.                                               11      report is not likely to add to very much. You are going
- 12   LORD JUSTICE LEVESON: Again, as long as that is broadly 12         to be able to do, one would hope, 90 or 95 per cent of
+ 12   LORD JUSTICE LEVESON: Again, as long as that is broadly    12      to be able to do, one would hope, 90 or 95 per cent of
  13     understood, then I don't want to create a structure      13      the job in terms of its quality in one year as you could
  14     which is formalistic, time consuming and potentially     14      in three years and the extra 10 per cent.
  15     difficult to operate.                                    15    LORD JUSTICE LEVESON: I might disagree with the
@@ -1539,15 +1539,15 @@
  25     paragraph 18 and subparagraph 2. There isn't or at       25      that fairness cannot be ignored or short circuited, but,
                            Page 105                                                                Page 107
 
-  1     least there wasn't intended to be because paragraph 15    1      again, we don't live in an ideal world and I do ask the
-  2     is dealing with the issue of whether witnesses will be    2      core participants to allow us a degree of latitude. But
-  3     asked to give oral evidence at all or to whether their    3      against that background we hope that the Inquiry will be
-  4     witness statements will simply be taken as read or        4      concluded within the timetable which has been ordained.
-  5     summarised. Paragraph 18 is designed to cater for the     5    LORD JUSTICE LEVESON: Yes. There was a submission that we
-  6     situation where the witness is giving oral evidence but   6      ought to change something we said in the protocols.
-  7     how is that oral evidence to be adduced? Because in       7    MR JAY: I suggested an agreed amendment to paragraph 15 of
-  8     some witnesses it is being suggested the evidence will    8      the documents protocol and paragraph 3(b) of the
-  9     be led in the -- if I can say the old-fashioned way --    9      assessor's protocol. I wasn't aware that there were any
+  1     least there wasn't intended to be because paragraph 15     1     again, we don't live in an ideal world and I do ask the
+  2     is dealing with the issue of whether witnesses will be     2     core participants to allow us a degree of latitude. But
+  3     asked to give oral evidence at all or to whether their     3     against that background we hope that the Inquiry will be
+  4     witness statements will simply be taken as read or         4     concluded within the timetable which has been ordained.
+  5     summarised. Paragraph 18 is designed to cater for the      5   LORD JUSTICE LEVESON: Yes. There was a submission that we
+  6     situation where the witness is giving oral evidence but    6     ought to change something we said in the protocols.
+  7     how is that oral evidence to be adduced? Because in        7   MR JAY: I suggested an agreed amendment to paragraph 15 of
+  8     some witnesses it is being suggested the evidence will     8     the documents protocol and paragraph 3(b) of the
+  9     be led in the -- if I can say the old-fashioned way --     9     assessor's protocol. I wasn't aware that there were any
  10     with other witnesses, much of their evidence will be      10     other points which hadn't been covered. We don't think
  11     taken as read and that will be a judgment call for        11     there were any other points.
  12     counsel to the Inquiry.                                   12 LORD JUSTICE LEVESON: Do you say anything about the
@@ -1686,15 +1686,15 @@
 (+44) 207 404 1400                                                                                                 London EC4A 2DY
                                                 Leveson Inquiry Initial Hearing                                                4 October 2011
 
-  1  the matters to which the Inquiry relates or whether you       1      people who complain they have been libelled in court
-  2  have a significant interest in the important aspect of        2      proceedings, but the problem about that is what is said
-  3  the matters to which the Inquiry relates, because I am        3      in court is privileged -- anyway.
-  4  not considering specifically the law of libel, as far as      4    UNKNOWN SPEAKER: I don't mean that -- the privilege.
-  5  I'm aware, or that you are going to be criticised. I am       5      I mean the outcome of the proceedings, certainly with no
-  6  sure you are not going to be criticised in the course of      6      right of reply.
-  7  my proceedings.                                               7    LORD JUSTICE LEVESON: But my Inquiry is rather different
-  8     So I understand why you are asking. I am just not          8      from that being conducted by the Department of Culture,
-  9  sure you fit; do you see?                                     9      Media and Sport Committee in the House of Commons and
+  1  the matters to which the Inquiry relates or whether you        1     people who complain they have been libelled in court
+  2  have a significant interest in the important aspect of         2     proceedings, but the problem about that is what is said
+  3  the matters to which the Inquiry relates, because I am         3     in court is privileged -- anyway.
+  4  not considering specifically the law of libel, as far as       4   UNKNOWN SPEAKER: I don't mean that -- the privilege.
+  5  I'm aware, or that you are going to be criticised. I am        5     I mean the outcome of the proceedings, certainly with no
+  6  sure you are not going to be criticised in the course of       6     right of reply.
+  7  my proceedings.                                                7   LORD JUSTICE LEVESON: But my Inquiry is rather different
+  8     So I understand why you are asking. I am just not           8     from that being conducted by the Department of Culture,
+  9  sure you fit; do you see?                                      9     Media and Sport Committee in the House of Commons and
  10 UNKNOWN SPEAKER: No, I see, because I know this all started    10     I am absolutely -- I have taken over a great part of
  11  as a phone hacking issue. However, as, sort of, the           11     their remit but not necessarily all of it.
  12  Press Standards Inquiry from the culture minister of the      12 UNKNOWN SPEAKER: But I must say from module 1 that you were
@@ -1772,7 +1772,7 @@
                             Page 121                                                               Page 123
 
   1     my application?                                            1      want to or you can stay or whatever.
-  2   LORD JUSTICE LEVESON: Yes. I will make a ruling about your 2      UNKNOWN SPEAKER: I always like to hear what Mr Sherborne
+  2   LORD JUSTICE LEVESON: Yes. I will make a ruling about your   2    UNKNOWN SPEAKER: I always like to hear what Mr Sherborne
   3     application. Not just yet because I will hear Mr Jay       3      has to say.
   4     first, but then I will make a ruling and then you can      4             Application by MR SHERBORNE
   5     deal with it as you think is appropriate.                  5    MR SHERBORNE: Sir, as I indicated I have an application on
@@ -1790,10 +1790,10 @@
  17   MR JAY: Yes, it is.                                         17    LORD JUSTICE LEVESON: Yes. What I think I will ask you to
  18   LORD JUSTICE LEVESON: Thank you. Do you want to say         18      do, Mr Sherborne, is to reduce in writing in a short
  19     anything else? I will give you the last word, you see.    19      letter the particular details touching upon those two
- 20   UNKNOWN SPEAKER: It is the last word? I know your team is 20        persons.
+ 20   UNKNOWN SPEAKER: It is the last word? I know your team is   20      persons.
  21     aware that my opponent has been Mr Sherborne. So I hope   21    MR SHERBORNE: I am more than happy to do so. We would have
  22     this isn't influencing --                                 22      done it in advance, except this, unfortunately, had to
- 23   LORD JUSTICE LEVESON: No, it is not at all. All I knew was 23       be done at very short notice, but I can deal with it in
+ 23   LORD JUSTICE LEVESON: No, it is not at all. All I knew was  23      be done at very short notice, but I can deal with it in
  24     that there was a reason why the team of core              24      writing.
  25     participants couldn't act for you. I didn't know why      25    LORD JUSTICE LEVESON: I am not being critical but I think
                             Page 122                                                               Page 124
@@ -2823,4 +2823,4 @@
 
 Merrill Legal Solutions                    www.merrillcorp/mls.com         8th Floor 165 Fleet Street
 (+44) 207 404 1400                                                              London EC4A 2DY
-
\ No newline at end of file
+
